Adds the Intrepid and it's Corresponding Files.

Doctor-Derp opened the pull request:
## About the Pull Request

Adds a lowpop ship the TGLV Intrepid! (And it's relevant files.)

**Thank you to the original Torch mappers, and baystation devs for giving me a template to work off of, as well as some examples to help me along the way!**

(Bottom Deck. W.I.P.)
![intrepid1-1-small](https://github.com/vlggms/tegustation-bay12/assets/53223414/77dfa1ba-0a41-48e8-8d96-dbcfbb25be8d)

(Top Deck. W.I.P.)
![intrepid2-1-small](https://github.com/vlggms/tegustation-bay12/assets/53223414/b8601e21-1b7e-4544-be88-665452f40768)

## Why It's Good For The Game

New ship! A lowpop one too, so 1 or 2 people don't have to deal with the 6-deck Torch all the time. Also variety is nice. 

If people want the Torch badly, or if we have enough population, we can switch. I didn't expect this to be a full Torch replacement.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Did you test it?

Yes- I'm testing it as we go along. It's still a work in progress.
<!--
Please describe if you ran local tests to ensure compilation. If that is not the case, please make it abundantly clear so a maintainer knows they need to run local checks.
Note that this can include own balancing/gameplay tests, but does not need to.
-->

## Changelog

:cl:
rscadd: Adds a new map!
maptweak: Adds a new map!
/:cl:

<!--
Common tags:
* rscadd - Adding a feature.
* rscdel - Removing a feature.
* tweak - Changing an existing feature.
* bugfix - Fixing an intended functionality that is not working, or correcting an oversight.
* maptweak - Changing something on a map, or adding a new away site. In 99% of cases, all map changes are maptweak.
* spellcheck - Spelling and grammar fixes.
Uncommon tags:
* admin - Adding, removing or changing administrative tools.
* balance - Changing an existing feature in such a way that it may broadly impact game balance; usually reserved for larger changes.
* soundadd - Adding new sounds, usually covered by rscadd unless you're only adding the sounds themselves.
* sounddel - Ditto as above with rscdel
* imageadd - Adding new icons; same situation as soundadd - usually you're adding something that uses these icons, so this isn't needed
* imagedel - Ditto as above.
* experiment - For experimental changes and tests that are intended to be temporary.
* wip - For works in progress. You probably won't get away with using this one.

Examples were changelog entries are optional/not typically required but encouraged:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation

As a courtesy, for ported PRs, please include if you have the blessing of the author. While not required, we encourage basic decency in porting others' work. It is also sufficient to note that the author has not expressed displeasure at the idea of their work getting ported.
Please refrain from porting works of authors that have expressed displeasure in having their work ported, thank you.
-->

Doctor-Derp committed:
Intrepid

Intrepid

Shadowtail117 reviewed (commented):
CI is currently failing on the following, for reference:
```Map key 'intrepid' is present in the repository, but is not listed in .github/workflows/test.yml!```

```
	test: running(fail) "maps contain no step_[xy]"
	test: running(fail) "maps contain no layer adjustments"
maps/intrepid/intrepid1.dmm:"Ri" = (/obj/item/storage/mirror{pixel_y = 29},/obj/structure/hygiene/sink{layer = 3; pixel_y = 23},/turf/simulated/floor/tiled/dark/monotile,/area/space)
```

```124 to_world uses (expecting exactly 117)```


```
## ERROR: Cannot read null.x - sound_player.dm - 229
## ERROR: proc name: PrivUpdateListenerLoc (/datum/sound_token/proc/PrivUpdateListenerLoc)
## ERROR: Cannot read null.x - sound_player.dm - 229
## ERROR: proc name: PrivUpdateListenerLoc (/datum/sound_token/proc/PrivUpdateListenerLoc)
## ERROR: Cannot read null.x - sound_player.dm - 229
## ERROR: proc name: PrivUpdateListenerLoc (/datum/sound_token/proc/PrivUpdateListenerLoc)
```

EgorDinamit commented:
Will be helping out with this PR's code part.

EgorDinamit committed:
holomap area colors

Doctor-Derp committed:
adds emergency lights, fixes some access, and more

Doctor-Derp committed:
makes airlocks actually work

EgorDinamit committed:
fix marine suit

Doctor-Derp committed:
updoots

EgorDinamit committed:
updaterinos

EgorDinamit committed:
supply packs for marines, replace T12 with military rifle

EgorDinamit committed:
fix

EgorDinamit committed:
Blob fix

EgorDinamit committed:
Vines fix

EgorDinamit committed:
fix supply-pack

EgorDinamit committed:
up

Doctor-Derp committed:
makes muhreens better

EgorDinamit committed:
update & fix

Doctor-Derp committed:
fix fix fix

EgorDinamit committed:
stuff updated

Doctor-Derp committed:
brings the ship up to code

Doctor-Derp committed:
power issues fixed

EgorDinamit committed:
Fix combat engi and medic access; Adds blue and orange stripes to IDs.

Doctor-Derp committed:
Space Truck

Doctor-Derp committed:
general improvements

Doctor-Derp committed:
Oops, all docks! (Also some engine fixes)

Doctor-Derp committed:
this should fix the engines + fax machines

Doctor-Derp committed:
kills combustion engine

EgorDinamit committed:
SL access, serpentid jobs and map fixes

Doctor-Derp committed:
fix fix fix

Doctor-Derp committed:
fix hangar power

EgorDinamit committed:
Adds OBS shuttle to Intrepid away sites

EgorDinamit committed:
blob fix

EgorDinamit committed:
Landmark and disperser fixes

Doctor-Derp committed:
solar fixes and CC revamp

EgorDinamit committed:
update

EgorDinamit committed:
tweak and fix for ranks

EgorDinamit committed:
Don't encode desc

Doctor-Derp committed:
FIXES LIFTS!

Doctor-Derp committed:
Update intrepid1.dmm

EgorDinamit committed:
job mannequin tweak

Doctor-Derp committed:
Adds more paint, tweaks atmospherics.

Adds more paint, tweaks atmospherics.

EgorDinamit committed:
Shuttles require power

Doctor-Derp committed:
Removes crystal walls, makes Atlas non-dense

Commit:
expands chem, expands the bar

EgorDinamit committed:
Uses Terran flags and seals instead of SolGov

EgorDinamit committed:
loadouts

EgorDinamit committed:
Replace keycard auths with Intrepid version

Commit:
Minor improvements to bar, maints, and other areas

EgorDinamit committed:
fix medals

EgorDinamit committed:
Fix access string

EgorDinamit committed:
Engi roles fix

EgorDinamit committed:
Remove "ideal character age"

EgorDinamit committed:
Orange marines

EgorDinamit committed:
marine helmet tweaks

EgorDinamit committed:
Posters

EgorDinamit committed:
RCD update

EgorDinamit committed:
Changes drone fabricator tags

EgorDinamit committed:
Fix AI lawset

Commit:
An attempt to fix the Caduceus

EgorDinamit committed:
Enables bunker away site

EgorDinamit committed:
Map tweaks

More holopads, more ammo in sec lockers, replaces sec voidsuits to alt type(looks better) + re-adds arm/leg guards to marine room.

Commit:
Intrepid grows 2 tiles wider

EgorDinamit committed:
Intrepid system name changed

EgorDinamit committed:
Map fixes

EgorDinamit committed:
Supply pack tweaks

Commit:
You want a morgue, YOU'LL GET A GODDAMN MORGUE.

EgorDinamit committed:
Mapping fixes

EgorDinamit committed:
Adds radio beacons to Intrepid bridge and Pandora

Commit:
S'more fixes

Commit:
the Archive

Commit:
archive fix

EgorDinamit committed:
Lift fix(?)

Commit:
mime secret, request consoles, shuttle/archive fix

Commit:
marine ladder among other things

EgorDinamit committed:
Singularity ring enabled

EgorDinamit committed:
Tiny fixes

EgorDinamit committed:
Singularity ring fixes

EgorDinamit committed:
Allow medical & scientist jobs to aliens

EgorDinamit committed:
Removes mentions of old name(Ion) from ID cards

EgorDinamit committed:
Minor tweaks

EgorDinamit committed:
Tweaks to marine guns

Default marine weapon is now T18. Added T18 and AK47 to supply packs.

EgorDinamit committed:
oops

EgorDinamit committed:
Taur planet added to available home planets

EgorDinamit committed:
Event fixes

EgorDinamit committed:
Virology + some fixes

EgorDinamit committed:
Antag restritected jobs

EgorDinamit committed:
Virology access

EgorDinamit committed:
Various shuttle fixes

EgorDinamit committed:
Merchant ship docking area fix

EgorDinamit committed:
Fix some elevator stuff

EgorDinamit committed:
Map fixes

EgorDinamit committed:
Fix cargo lift

EgorDinamit committed:
Some tweaks

EgorDinamit committed:
Remote weapons on Intrepid

EgorDinamit committed:
Pods and safe rooms should work now, I guess

EgorDinamit committed:
Fixes pods?

EgorDinamit committed:
Pods, pods, pods

EgorDinamit committed:
Fix

EgorDinamit committed:
Adds lights to pods and centcom recovery shuttle

EgorDinamit committed:
Please work

EgorDinamit committed:
Final touches

EgorDinamit committed:
A single light bulb

EgorDinamit committed:
fix unit test

EgorDinamit committed:
Some changes

EgorDinamit committed:
Tiny detail!

EgorDinamit committed:
Third is the loneliest pod.

EgorDinamit committed:
Tiny tweaks

EgorDinamit committed:
More medals! More loadout stuff!

EgorDinamit committed:
Fix loadout age requirement display problem

EgorDinamit committed:
Advanced first aid skill badge loadout needs trained medical

EgorDinamit committed:
Intrepid FTL effect

EgorDinamit committed:
Adds bluespace drive to the map

EgorDinamit committed:
Actually adds loadout age lock

EgorDinamit committed:
a

EgorDinamit committed:
Submap/other ranks added

EgorDinamit committed:
Assorted tweaks

EgorDinamit committed:
Gives terran banner an icon

Commit:
changes

Commit:
WOO

Commit:
fixes

EgorDinamit committed:
Fix OBS ranks

EgorDinamit committed:
Fix OBS rank/branch on spawn

EgorDinamit committed:
Shuttle APCs upgraded

They now have super cells and rapid batteries

EgorDinamit committed:
Some fixes

EgorDinamit committed:
More fixes

EgorDinamit committed:
Fix wrong access text

EgorDinamit committed:
Fix explo banners.

EgorDinamit committed:
Fix merchant dock

EgorDinamit committed:
Planet claims round-end report.

Commit:
big changes

EgorDinamit committed:
Removes tint from marine helmets

EgorDinamit committed:
Removes backup file

Backup is what github is for

Commit:
fix

EgorDinamit committed:
Cable fix

EgorDinamit committed:
Restores AI whitelist

Commit:
hopefully this fixes more stuff

Commit:
Engineering hallway makeover

EgorDinamit committed:
Terran coffee cup

EgorDinamit committed:
Replaces CoS rig with a void suit, some poster changes

Commit:
damned be the north and west pipes

Commit:
Firing Range

Commit:
megacommit

Commit:
intrepid_unit_testing.dm changes

Commit:
I guess solars just doesn't want to work

Commit:
singularity

Commit:
more fixes more fixes more fixes

Commit:
goodbye, stardust

EgorDinamit committed:
Fix pipes in chemistry, add light replacers

EgorDinamit committed:
Adds new homeplanets to the available list

EgorDinamit committed:
Fix some mapping issues

EgorDinamit committed:
Direction signs

Commit:
tweaks sign placement, fixes stardust access, adds light switches

EgorDinamit committed:
English is default language

EgorDinamit committed:
Minor fix

EgorDinamit committed:
English papers

EgorDinamit committed:
Fix overloaded lighting trait

EgorDinamit committed:
Shuttles use SMES instead of cell racks

EgorDinamit committed:
Removes unused access, adds military access region

EgorDinamit committed:
Fix

EgorDinamit committed:
Fixes req access on braces

EgorDinamit committed:
Adds unit test safe turf landmark

This fixes a lot of unit tests

EgorDinamit committed:
Fixes solars

EgorDinamit committed:
Fixes meeting room windows

Commit:
fixes research virology, removes some shutoffs

EgorDinamit committed:
Places singularity gen in its place

EgorDinamit committed:
Increases the range of window tint button

FIX!!!

EgorDinamit committed:
All silicons start with English as default language

EgorDinamit committed:
Prevents Intrepid's "roof" from being deleted by arriving shuttles

EgorDinamit committed:
Adds ATMs near docking ports

EgorDinamit committed:
Moves jetpacks to the top, so they can be more noticeable

EgorDinamit committed:
Places OBS suit storage units on their shuttle

Commit:
replaces some decals, removes some shutoffs

Commit:
AI core and Engineering overhaul

a textbook lesson in the mapping of secure multi-z level areas

EgorDinamit committed:
Moves mapping helper to above doors layer

Commit:
please be kind github

Commit:
all shutoff valves SLAIN

Commit:
one step closer to merging

EgorDinamit committed:
Adds design datum for remote weapon

EgorDinamit committed:
Fixes dominion tech color & design name

Commit:
paint and psi office

EgorDinamit committed:
Security states

EgorDinamit committed:
Tiny fix to security psi states

EgorDinamit committed:
Makes atmospherics have proper access reqs

Commit:
downgrades solars, fixes atmos

EgorDinamit committed:
uncomment

EgorDinamit committed:
Applies whitelist of factions to several Intrepid jobs

EgorDinamit committed:
Fixes sci jobs culture whitelist

EgorDinamit committed:
Adds reborn christianity to the list of available cultures

EgorDinamit committed:
Fixes leaking pipes

EgorDinamit committed:
Adds missing ammo types to autolathe, gives revolvers to XO and Senator, other stuff.

CO gets a heavy revolver, XO gets holdout revolver, Senator gets normal military revolver.
Autolathe now has missing ammo magazines. Senator's door now has unique-ish design.

EgorDinamit committed:
Places holsters in revolver closets

EgorDinamit committed:
Cargo rework compliance

EgorDinamit committed:
More ATMs

EgorDinamit committed:
Trade faction

EgorDinamit committed:
Supply consoles placement

EgorDinamit committed:
Marine backpacks

EgorDinamit committed:
Fixes wrong speedloader type

EgorDinamit committed:
Supply access

Commit:
cargo beautification

EgorDinamit committed:
Range of receiving beacon is set to 1

EgorDinamit committed:
Critical APCs/SMES tweaks

Commit:
replaces the pandora, adds telescreens

Commit:
MODIFIES STEP_TRIGGERS.dm + TRANSIT.dmm

EgorDinamit committed:
Marine lockers are more resilient

Commit:
reworks security, adds account, tweaks science

EgorDinamit committed:
Fixes small issues in security

Commit:
cryogenics redesign + pandora airlock fix + secfix

Commit:
brig access tweaks

EgorDinamit committed:
Fixes leaking pipes

EgorDinamit committed:
ERC poster

EgorDinamit committed:
Poster name fixes

EgorDinamit committed:
Job culture restrictions reduced

EgorDinamit committed:
Envelope with consul orders

Commit:
more fixes more fixes more fixes

EgorDinamit committed:
Trusted player jobs, maximum character age

This adds marine veteran role, the first trusted locked role

EgorDinamit committed:
Adds marine veteran to the job list

EgorDinamit commented:
Long overdue.
